uberspace/backup_cron

```diff
@@ -0,0 +1 @@
+0 2 * * * /bin/bash ~/serlo-editor-as-lti-tool/uberspace/database_backup.sh
```

Comment:
Would you consider adding a comment here, which explains how often this runs?

Alternatively, a website like https://crontab.guru/ is helpful as well.

Reply:
We can add a comment once this will probably not change any more. For now, I linked crontab.guru